pkg/route/graph/analysis/analysis.go

```diff
@@ -86,3 +90,25 @@ func FindMissingTLSTerminationType(g osgraph.Graph, f osgraph.Namer) []osgraph.M
 
 	return markers
 }
+
+func FindPathBasedPassthroughRoutes(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
+	markers := []osgraph.Marker{}
+
+	for _, uncastRouteNode := range g.NodesByKind(routegraph.RouteNodeKind) {
+		routeNode := uncastRouteNode.(*routegraph.RouteNode)
+
+		if len(routeNode.Spec.Path) > 0 && routeNode.Spec.TLS != nil && routeNode.Spec.TLS.Termination == routeapi.TLSTerminationPassthrough {
```

Comment:
Question: Does this get more maintainable if you whitelist things that *can* use paths instead of things that can't?  I would probably whitelist instead of blacklist to make maintenance easier.